conduct.md

```diff
@@ -116,15 +119,21 @@ how long a decision will take to implement.
     * Report errors or bugs to the appropriate team members.
     * When a choice is disputed, use Github issue tracker and let team members
       share their opinion.
-    * :heavy_exclamation_mark: TODO: add any additional items in the A category for the decision making as a team.
+    * Defer to the Professor or Technical Leaders when unsure about tasks or implementation.
+    * Use efficient communication within and outside of teams, where ideas are clearly expressed in the shortest possible timeframe.
+    * Communicate with the appropriate team member about any changes that is needed to be done to any project.
+    * Stay respectful and open-minded when discussing decisions and ideas.
 
 * Unacceptable Conduct (U):
   * Members shall not
     * Let an individual team member or a project manager make all major decisions.
     * Make decisions without finding evidence showing that it would likely be beneficial.
     * Refuse to work on the decided option.
     * Make decisions that would affect another team without first notifying them.
-    * :heavy_exclamation_mark: TODO: add any additional items in the U category for the decision making as a team.
+    * Refuse to actively participate in group discussions or breakout rooms.
+    * Deploy any kind of decision that's against codes of ethic.
```

Comment:
I think it would be better to not use contraction in a formal document like this. namely, "that is" instead of "that's"